search/grounded-generation-playground/src/components/ui/about-page-content.tsx

```diff
@@ -0,0 +1,204 @@
+                  <p>
+                    Dynamic Retrieval can be enabled along with a threshold to only do a
+                    Google Search when we think it's needed. Fewer searches saves you cost
+                    and speeds up the response.
+                  </p>
+                </div>
+              </CardContent>
+            </Card>
+          )}
+          {vertexGrounding && (
+            <Card className="rounded-2xl bg-zinc-800 text-white h-full">
+              <CardContent>
+                <h3 className="my-2 flex items-center">
+                  <Icon type="vertex" className="mr-2 text-gray-400 h-6 w-6" />
+                  Vertex AI Search
+                </h3>
+                <p>Enter your own Vertex AI Search project path.</p>
+                <p>
+                  An example would look like:{' '}
+                  <code>
+                    projects/555555555555/locations/global/collections/default_collection/engines/VERTEX_AI_SEARCH_APP_ID/servingConfigs/default_search
+                  </code>
+                </p>
+              </CardContent>
+            </Card>
+          )}
+        </div>
+      </TabsContent>
+    </Tabs>
+  );
+};
+
+const makeCurl = (requestObj: RequestObject): string => {
+  const PROJECT_NUMBER = process.env.PROJECT_NUMBER;
+  const API_ENDPOINT = `https://discoveryengine.googleapis.com/v1alpha/projects/${PROJECT_NUMBER}/locations/global:streamGenerateGroundedContent`;
+  const requestStr = JSON.stringify(requestObj);
+  return `curl -X POST \\  
+-H "Authorization: Bearer $(gcloud auth print-access-token)" \\
+-H "Content-Type: application/json" \\
+"${API_ENDPOINT}" \\
+-d '${requestStr}'`;
+};
+
+const makeJs = (requestObj: RequestObject): string => {
+  const PROJECT_NUMBER = process.env.PROJECT_NUMBER;
+  const API_ENDPOINT = `https://discoveryengine.googleapis.com/v1alpha/projects/${PROJECT_NUMBER}/locations/global:streamGenerateGroundedContent`;
+  const requestStr = JSON.stringify(requestObj, null, 4);
+  return `
+const auth = new GoogleAuth({scopes: ['https://www.googleapis.com/auth/cloud-platform']});
+const client = await auth.getClient();
+const accessToken = await client.getAccessToken();
+
+const response = await fetch('${API_ENDPOINT}', {
+    method: 'POST',
+    headers: {
+        Authorization: 'Bearer \${accessToken.token}',
+        'Content-Type': 'application/json',
+    },
+    body: JSON.parse(\`[
+${requestStr}
+]\`), 
+});`;
+};
+
+const makePython = (requestObj: RequestObject): string => {
+  const PROJECT_NUMBER = process.env.PROJECT_NUMBER;
+  const API_ENDPOINT = `https://discoveryengine.googleapis.com/v1alpha/projects/${PROJECT_NUMBER}/locations/global:streamGenerateGroundedContent`;
+  const requestStr = JSON.stringify(requestObj, null, 4);
+  return `import requests
+import json
+import google.auth
+credentials, project = google.auth.default(scopes=['https://www.googleapis.com/auth/cloud-platform'])
+
```

Comment:
Add TODO to use Client Libraries when they are available.

Reply:
done